github.com/regen-network/regen-ledger/x/group/testdata/tx.pb.go

```diff
@@ -0,0 +1,327 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
@aaronc By running `make proto-gen` i have regenerated all protos inside redgen ledger. Do you wanna me to keep only testdata/tx.pb.go that has been moved from types/testutil and delete other regenerated *.pb.go files or it looks okay to you?